package/src/custom/tests/School/unit-php/phpunit.xml.dist

```diff
@@ -0,0 +1,56 @@
+<!--
+/*
+ * Your installation or use of this SugarCRM file is subject to the applicable
+ * terms available at
+ * http://support.sugarcrm.com/Resources/Master_Subscription_Agreements/.
+ * If you do not agree to all of the applicable terms or do not have the
+ * authority to bind the entity as an authorized representative, then do not
+ * install or use this SugarCRM file.
+ *
+ * Copyright (C) SugarCRM Inc. All rights reserved.
+ */
+-->
+
+<phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:noNamespaceSchemaLocation="http://schema.phpunit.de/4.1/phpunit.xsd"
+         backupGlobals="false"
+         backupStaticAttributes="false"
+         colors="true"
+         convertErrorsToExceptions="true"
+         convertNoticesToExceptions="true"
+         convertWarningsToExceptions="true"
+         processIsolation="false"
+         stopOnFailure="false"
+         bootstrap="../../../../tests/unit-php/TestBootstrap.php"
+         beStrictAboutOutputDuringTests="true"
+         beStrictAboutTestsThatDoNotTestAnything="true"
+         beStrictAboutTestSize="true"
+         beStrictAboutCoversAnnotation="false"
+>
+
+    <php>
+        <ini name="intl.default_locale" value="en"/>
+        <ini name="intl.error_level" value="0"/>
+        <ini name="memory_limit" value="-1"/>
+    </php>
+    <testsuites>
+        <testsuite name="all">
+            <directory>./</directory>
+            <directory>modules/</directory>
```

Comment:
Doesn't `./` includes `modules/`? It should be recursive. If you got rid of this second line, then would it still run everything?

Reply:
I removed it and the tests still ran.  I copied it out of the regular Sugar phpunit.xml.dist, so it's odd they have it there as well.

Reply:
I agree. I don't quite know why it was added in the first place. If you wanted to create a test suite that runs only tests that validate the logic hook handlers, then you might have a "logic hook" suite that specifies only the files and/or directories where those tests exist. But I don't think you need to worry about that yet. Running everything is fast.